pkg/aws/snow.go

```diff
@@ -0,0 +1,56 @@
+package aws
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+)
+
+const (
+	snowEC2Port = 8243
+)
+
+func snowEndpoint(deviceIP string) *ServiceEndpoint {
+	return &ServiceEndpoint{
+		ServiceID:     "EC2",
+		SigningRegion: "snow",
+		URL:           fmt.Sprintf("https://%s:%d", deviceIP, snowEC2Port),
+	}
+}
+
+func WithCustomCABundleFile(certsFile string) config.LoadOptionsFunc {
```

Comment:
For consistency on this package API, you probably want to return an `AwsConfigOpt`. The compiler shouldn't complain because its an alias but if it does you can wrap the return with `AwsConfigOpt()` to perform a type conversion.